libkbfs/reporter_kbpki.go

```diff
@@ -267,6 +283,15 @@ func (r *ReporterKBPKI) send(ctx context.Context) {
 				r.log.CDebugf(ctx, "ReporterDaemon: error sending "+
 					"notification: %s", err)
 			}
+		case path, ok := <-r.notifyPathBuffer:
+			if !ok {
+				return
+			}
+			if err := r.config.KeybaseService().NotifyPathUpdated(
```

Comment:
It seems it makes sense to make this affected by suppression too. WDYT?

Reply:
Also, it seems this notification would include changes from the local device too. So perhaps we can remove the suppression from `Notify`.

Reply:
I added suppression logic for the path updates, but I didn't want to mess with the `Notify` logic since I have no idea if the frontend is still using them or if you've switched completely over to path notifications yet (I'm guessing not, since I haven't even turned them on yet!).